.github/workflows/release.yml

```diff
@@ -29,6 +29,7 @@ jobs:
         nox -e build
 
     - name: Publish package distributions to PyPI
+      if: github.event.release.released
```

Comment:
```suggestion
```

If `types: [released]` works (from other suggestion), we can remove this if conditional.

Reply:
Done